src/Date.php

```diff
@@ -127,8 +127,7 @@ public function isValid($value)
      */
     protected function convertToDateTime($param, $addErrors = true)
     {
-        // @TODO: when minimum dependency will be PHP 5.5, we can only keep check against DateTimeInterface
-        if ($param instanceof DateTime || $param instanceof DateTimeInterface) {
+        if ($param instanceof DateTimeInterface) {
```

Comment:
As per discussion with the internal folks (sigh), this should probably be:

```php
if ($param instanceof DateTime || $param instanceof DateTimeImmutable) {
```

According to internal folks, `DateTimeInterface` is not to be used.